src/server/db/booking-schema.ts

```diff
@@ -0,0 +1,30 @@
+import { relations } from "drizzle-orm";
+import { pgTable, serial, text, timestamp } from "drizzle-orm/pg-core";
+import { user } from "./auth-schema";
+
+export const bookings = pgTable("bookings", {
+  id: serial("id").primaryKey(),
+  title: text("title").notNull(),
+  pickupLocation: text("pickup_location").notNull(),
+  dropoffLocation: text("dropoff_location").notNull(),
+  purpose: text("purpose"),
+  passengerInfo: text("passengerInfo").notNull(),
+  status: text("status", { enum: ["incomplete", "completed", "in-progress"] })
+    .notNull()
+    .default("incomplete"),
+  driverId: text("driver_id")
+    .notNull()
+    .references(() => user.id),
+  createdAt: timestamp("created_at").defaultNow().notNull(),
+  updatedAt: timestamp("updated_at")
+    .defaultNow()
+    .$onUpdate(() => /* @__PURE__ */ new Date())
+    .notNull(),
```

Comment:
for the booking, lets also add createdby and updatedby

```suggestion
createdBy: text("created_by").references(() => user.id),
updatedBy: text("updated_by").references(() => user.id),
```